guides/common/modules/proc_acd_installation.adoc

```diff
@@ -0,0 +1,43 @@
+[id="{context}_installation"]
+= Installation
+
+You need to install both the `foreman_acd` and `smart_proxy_acd` plugins to use ACD.
+
+This describes the package installation on CentOS 7, Red Hat Enterprise Linux 7, and Oracle Linux 7.
+
```

Comment:
It would worth explaining that this plugin will work both for Foreman and Katello installations and making an assumption that Foreman (and/or Katello) is installed already - probably referring to the installation guides from here.

Reply:
I've linked both guides and added a warning for Debian and Ubuntu users that ACD is currently untested on Debian based Foreman installations.